src/services/bags.ts

```diff
@@ -95,10 +95,10 @@ export async function bagsListCheck(api: ApiPromise, account: KeyringPair, sendT
 	assert.deepEqual(counter, counterOnchain.toNumber());
 	assert.deepEqual(counter, nominatorsOnChain.toNumber());
 
-	const txsInner = needRebag.map((who) => api.tx.bagsList.rebag(who)).slice(count);
+	const txsInner = needRebag.map((who) => api.tx.bagsList.rebag(who)).slice(0, count);
```

Comment:
This is the most important change. Prior we started the slice from `count` and included everything until the end. Now we start at zero and include everything until `count`